### PR TITLE
To use phinx for unit testing, sometimes is necessary to run not just…

### PR DESCRIPTION
**Pull request by cauelorenzato**
… the migrations, but the seeds as well. This commit add a new method getSeed that runs the seeds.

One commit only as you asked.

### CODE REVIEW

File: src/Phinx/Wrapper/TextWrapper.php [hunk 1]
```diff
@@ -125,6 +125,29 @@ public function getMigrate($env = null, $target = null)
     }
 
     /**
+     * Returns the output from running the "seed:run" command.
+     * @param  string $env environment name (optional)
+     * @param  string $target target version (optional)
+     * @return string
+     */
+    public function getSeed($env = null, $target = null, $seed = null)
```
Comment (rquadling) [inline src/Phinx/Wrapper/TextWrapper.php]: Please add $seed to docblock

Comment (cauelorenzato) [inline src/Phinx/Wrapper/TextWrapper.php]: Done!

File: src/Phinx/Wrapper/TextWrapper.php [hunk 2]
```diff
@@ -125,6 +125,30 @@ public function getMigrate($env = null, $target = null)
     }
 
     /**
+     * Returns the output from running the "seed:run" command.
+     * @param  string $env environment name (optional)
```
Comment (rquadling) [inline src/Phinx/Wrapper/TextWrapper.php]: Can you use `string|null` as the type and drop `(optional)` please.

Comment (cauelorenzato) [inline src/Phinx/Wrapper/TextWrapper.php]: of course, done now.

**Comment by rquadling** [comment]
Thank you.

**Pull request merged**